Source/WebCore/layout/formattingContexts/inline/InlineFormattingContext.h

```diff
@@ -62,6 +62,7 @@ class InlineFormattingContext {
     std::pair<LayoutUnit, LayoutUnit> minimumMaximumContentSize(const InlineDamage* = nullptr);
     LayoutUnit minimumContentSize(const InlineDamage* = nullptr);
     LayoutUnit maximumContentSize(const InlineDamage* = nullptr);
+    std::optional<size_t> numberOfVisibleLinesAllowed() const;
```

Comment:
this should somehow reference to line-clamp to make it clear where this constrain comes from